controllers/ppo/ppo_deploy.py

```diff
@@ -0,0 +1,68 @@
+from enum import Enum, auto  # noqa: D100
+from typing import Any, Dict
+
+import numpy as np
+
+from lsy_drone_racing.command import Command
+
+
+class DroneState(Enum):
+    """Class to define the Drone State Machine states."""
+
+    TAKEOFF = auto()
+    POLICY_CONTROL = auto()
+    NOTIFY_SETPOINT_STOP = auto()
+    GOTO = auto()
+    LAND = auto()
+    FINISHED = auto()
+    NONE = auto()
+
+
+class StateMachine:
```

Comment:
Just as a preference, I prefer a more specific name like DroneStateMachine or similar so that it is cleaar that StateMachine is not a general StateMachine(like a Base class) but just specific to this problem. This is only a preference